smartseq2_single_sample/hca-dcp/wrapper_ss2_single_sample.wdl

```diff
@@ -0,0 +1,40 @@
+import "prepare_and_analyze_ss2_single_sample.wdl" as prep_and_analyze
```

Comment:
I re-read the description you wrote in the PR and readme comment ("do I need this?") and I get a sense that there are multiple reasons services calling different wrappers. I wanted to confirm that the prep_and_analyze wdl is expected to be run in multiple different services, and is generally be useful by itself, outside of the context of this calling wrapper.  

If not, I'd prefer to have `prepare_and_analyze_ss3_single_sample.wdl` broken apart and brought directly into this wrapper (3 imports of workflows instead of 1 workflow and 1 workflow wrapper). This would reduce the number of levels and simplify the logic that needs to be followed to understand what's being called.

Reply:
I think that's a good suggestion. I initially thought that people might want to reuse that prepare and analyze wdl but now can't think of a good reason why someone would want to - they'd be running the same analysis on the same data as the secondary analysis service does. Best to keep it simple for now and add that layer at a later time if we find a strong use case.